fivetran_provider_async/operators.py

```diff
@@ -67,18 +72,19 @@ def execute(self, context: Dict[str, Any]) -> None:
         hook.prep_connector(self.connector_id, self.schedule_type)
         hook.start_fivetran_sync(self.connector_id)
 
-        # Defer and poll the sync status on the Triggerer
-        self.defer(
-            timeout=self.execution_timeout,
-            trigger=FivetranTrigger(
-                task_id=self.task_id,
-                fivetran_conn_id=self.fivetran_conn_id,
-                connector_id=self.connector_id,
-                poke_interval=self.poll_frequency,
-                reschedule_wait_time=self.reschedule_wait_time,
-            ),
-            method_name="execute_complete",
-        )
+        if self.deferrable:
+            # Defer and poll the sync status on the Triggerer
+            self.defer(
+                timeout=self.execution_timeout,
+                trigger=FivetranTrigger(
+                    task_id=self.task_id,
+                    fivetran_conn_id=self.fivetran_conn_id,
+                    connector_id=self.connector_id,
+                    poke_interval=self.poll_frequency,
+                    reschedule_wait_time=self.reschedule_wait_time,
+                ),
+                method_name="execute_complete",
+            )
```

Comment:
Are we going to add sync logic?

Reply:
i think the current implementation only has async logic